pom.xml

```diff
@@ -584,14 +589,15 @@
         <imp.pkg.version.javax.servlet>[2.6.0, 3.0.0)</imp.pkg.version.javax.servlet>
         <apache.felix.scr.ds.annotations.version>1.2.4</apache.felix.scr.ds.annotations.version>
         <equinox.osgi.services.version>3.5.100.v20160504-1419</equinox.osgi.services.version>
+        <identity.data.publisher.authentication.version>5.1.9-SNAPSHOT</identity.data.publisher.authentication.version>
 
         <!--Carbon Kernel Version-->
         <carbon.kernel.version>4.4.17</carbon.kernel.version>
         <carbon.kernel.package.import.version.range>[4.4.0, 5.0.0)</carbon.kernel.package.import.version.range>
         <carbon.user.api.imp.pkg.version.range>[1.0.1, 2.0.0)</carbon.user.api.imp.pkg.version.range>
 
         <!--Carbon Identity Framework Version-->
-        <carbon.identity.framework.version>5.11.56</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.11.178-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
remove snapshot dependency

Reply:
merging PR wso2/carbon-identity-framework#1525 for update this with release version